test/helpers_test.go

```diff
@@ -161,7 +161,7 @@ func (d dirStatAsStringer) String() string {
 type srOpt func(*gwclient.SolveRequest)
 
 func newSolveRequest(opts ...srOpt) gwclient.SolveRequest {
-	sr := gwclient.SolveRequest{}
+	sr := gwclient.SolveRequest{Evaluate: true}
```

Comment:
Why was the `Evaluate` flag changed here?

Reply:
It really needs to be default because it can easily be missed and cause a test to pass unexpectedly.
`Evaluate` unlazy's the request, basically.

If we do a solve and don't interact with the returned state or export it somewhere, etc, it doesn't actually do the operations.

Reply:
Ah got it, that makes sense. Thanks for clarifying